application/templates/cms/measure_page_versions.html

```diff
@@ -39,7 +39,7 @@
 
     <div class='grid-row'>
         <div class="column-two-thirds">
-            <h1 class='heading-large'>Version history</h1>
+            <h1 class='heading-large'>Version history <div class="visually-hidden">of</div> <div class="heading-secondary">{{ measures[0].title }}</div></h1>
```

Comment:
Using the title from the first version of the array works (gets us the most recent title), but maybe it'd be good to somehow explicitly set this as being the 'latest' title somehow (eg by calling the `latest_version` method?) That way if we decided to reorder the list of versions (or only show the most recent x and paginate) it'd still work?

Could you also split this onto multiple lines for readability?